model/packed_array_typespec.yaml

```diff
@@ -36,18 +36,18 @@
     vpi: vpiRightRange
     type: expr
     card: 1
-  - group_ref: elem_typespec
+  - class_ref: elem_typespec
     name: elem typespec
     vpi: vpiElemTypespec
-    type: enum_struct_union_packed_array_typespec_group
```

Comment:
We become too permissive here. This group (like other groups) it meant to provide a runtime check that we don't construct an illegal model by checking that only the subset of classes are allowed to be used in this relation.